### ISSUE TITLE
download percentage > 100%

### ISSUE THREAD

**Issue by pohly** [clearlinux/swupd-client#957]
**Describe the bug**

While running "swupd update", swupd starts a "download of remaining update content". The progress for that step goes from 0% to 100% and then proceeds to 3766%. This also produces a lot of output.

```
Statistics for going from version 29940 to version 29970:

    changed bundles   : 4
    new bundles       : 0
    deleted bundles   : 0

    changed files     : 130
    new files         : 2
    deleted files     : 0

Starting download of remaining update content. This may take a while...
	...0%
	...1%
...
	...97%
	...100%
...
	...3766%
Staging file content
...
```

The update itself is successful.

**To Reproduce**

```shell
docker run clearlinux@sha256:b8e5d3b2576eb6d868f8d52e401f678c873264d349e469637f98ee2adf7b33d4 swupd update -m 29970
```

**Expected behavior**

I expect it to never go beyond 100%. If for whatever reason the original estimate was wrong and it does reach a higher value, it should print a warning and stop printing more percentages because they are useless.

**Environment (please complete the following information):**
 - Clear Linux OS Version: 29940, swupd client 3.20.0

**Comment by castulo** [comment]
Looks like this is a duplicate of bug #939 and this issue has already been fixed with #943 . It should be available in the next swupd release.

**Comment by castulo** [comment]
This bug is fixed in Clear Linux 30400, I'm closing this issue.
